libcudacxx/include/cuda/std/detail/libcxx/include/type_traits

```diff
@@ -415,13 +415,17 @@ namespace std
 
 */
 #ifndef __cuda_std__
-#include <__config>
-#else
-#ifndef _LIBCUDACXX_COMPILER_NVRTC
-#include <type_traits>
```

Comment:
question: this code dissapeared, is this intentional?

Reply:
I believe this is an old remnant of our structured bindings support that has since been moved to its own header